core/src/Kokkos_View.hpp

```diff
@@ -1618,7 +1618,7 @@ class View : public ViewTraits<DataType, Properties...> {
       : View(arg_prop,
              typename traits::array_layout(arg_N0, arg_N1, arg_N2, arg_N3,
                                            arg_N4, arg_N5, arg_N6, arg_N7)) {
-#ifdef KOKKOS_ACTIVE_EXECUTION_MEMORY_SPACE_HOST
+#if KOKKOS_IMPL_IF_ON_HOST
```

Comment:
??? How is this supposed to work?  Why `#if`?
https://github.com/kokkos/kokkos/blob/b3e6acd68c1874d59135a689704224fdf3f15ce1/core/src/impl/Kokkos_SharedAlloc.hpp#L51-L69

Reply:
Also https://github.com/kokkos/kokkos/blob/b3e6acd68c1874d59135a689704224fdf3f15ce1/core/src/impl/Kokkos_SharedAlloc.hpp#L519